Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17 as build
+FROM golang:1.18 as build
```

Comment:
Let's take a note that the next release needs to be a major upgrade because this might be a breaking change for some users.